wger/settings.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'wger/database.sqlite',
+        'USER': '',
+        'PASSWORD': '',
+        'HOST': '',
+        'PORT': '',
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = '_p$6&m_k9x&kt)a1%@1bdp3m0mi%ss51(haazs9@zpw01!2y8k'
+
+# Your reCaptcha keys
+RECAPTCHA_PUBLIC_KEY = ''
+RECAPTCHA_PRIVATE_KEY = ''
+NOCAPTCHA = True
+
+# The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
+# This is needed for uploaded files and images (exercise images, etc.) to be
+# properly served.
+SITE_URL = 'http://localhost:8000'
+
+# Path to uploaded files
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+MEDIA_ROOT = '/Users/robertndungu/.local/share/wger/media'
```

Comment:
this is hardcoded to your machine ... will cause issues in the future

Reply:
I have updated the path, to use a path in heroku.